www/include/monitoring/downtime/template/AddDowntime.ihtml

```diff
@@ -154,5 +154,37 @@
 
     initDatepicker("datepicker", "mm/dd/yy", "0");
 
+    function updateEndTime() {
```

Comment:
As this issue also impact other pages (ie: recurrent) or widgets' popin, you need to move this new method in :
`www/include/common/javascript/datepicker/localizedDatepicker.js` and call it on every pages using an endDate's datepicker